server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1302_create_table_ce_task_input.rb

```diff
@@ -21,15 +21,15 @@
 #
 # SonarQube 6.1
 #
-class CreateTableCeTaskData < ActiveRecord::Migration
+class CreateTableCeTaskInput < ActiveRecord::Migration
```

Comment:
now that it's been deployed to Dory, indeed we can merge the two migrations
we couldn't before :)